labels/de-DE.json

```diff
@@ -500,5 +500,6 @@
   "width": "Breite",
   "wrongCountryForType": "Bitte wählen Sie ein Land für diesen Kontakt Typ unterstützt",
   "yourAccount": "Ihr Konto",
-  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>."
+  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>.",
+  "licensePlateLabel":"Nummernschild # (OPTIONAL)"
```

Comment:
This file is sorted alphabetically. Add the new field to the right place. Don't add to the end.